webapp/src/components/molecules/UserDropDown/UserDropDown.tsx

```diff
@@ -20,68 +19,64 @@ export interface User {
 }
 
 export default function UserDropDown(props: User) {
-  const [loginTxt, setLoginTxt] = useState('Login');
-
   const navigate = useNavigate();
-
+  const { isAuthenticated, logout, loginWithRedirect } = useAuth0();
   const navigateTo = (path: string) => {
     navigate(path);
   };
 
+  console.log(navigate, 'inside UserDropDown');
+
   const menu = useRef<TieredMenu>(null);
 
-  const loginMenuItems = [
+  const items = [
+    {
+      label: 'Profile',
+      icon: 'pi pi-user',
+      command: () => {
+        navigateTo(`/profile/${props.user?.name}`);
+      },
+    },
     {
       label: 'Create Account',
       icon: 'pi pi-fw pi-user-edit',
-      // TODO Create Account Component isn't created yet but below is an option be used to navigate the user to that page
-      // command: () => {
-      //   navigateTo('/create-account');
-      // }
+    },
```

Comment:
Probably don't want Create Account in this list
![image](https://user-images.githubusercontent.com/485809/150239161-1ac39787-edc0-47b6-84a6-4a8a135453fd.png)